crates/ef-testing/tests/tests.rs

```diff
@@ -127,8 +127,8 @@ mod blockchain_tests {
     blockchain_tests!(st_transition_test, stTransitionTest);
     blockchain_tests!(st_wallet_test, stWalletTest);
     blockchain_tests!(st_zero_calls_revert, stZeroCallsRevert);
+    blockchain_tests!(st_zero_knowledge, stZeroKnowledge);
+    blockchain_tests!(st_zero_knowledge2, stZeroKnowledge2);
```

Comment:
Could you move the order of these tests to be in the same order as GeneralTests folder?